README.md

```diff
@@ -52,30 +52,71 @@ yet compatible with the golang BOSH v2.
 Required to create certificates. Requires golang.
 ```sh
 go get github.com/square/certstrap
+cd $GOPATH/src/github.com/square/certstrap
+go build
+go install
+```
+
+### stampy
+
+[TODO] What is it needed for?
```

Comment:
It's used to write timing information to a file, to (someday) look at it and figure out how we can make things go faster.  Maybe we should turn that into an optional dependency? (Not in this PR, of course)